fe/fe-core/src/main/java/com/starrocks/http/rest/RestBaseResult.java

```diff
@@ -81,8 +89,7 @@ public static RestBaseResult getOk() {
 
     @Legacy
     public String toJson() {
-        Gson gson = new Gson();
-        return gson.toJson(this);
+        return GSON.toJson(this);
```

Comment:
should not use GSON. the GSON object from GsonUtils has a lot of customization, may not be applicable to this class. If willing to save the frequent `new Gson()`, consider have a static Gson object in this class.